sopel/tools/target.py

```diff
@@ -19,11 +19,14 @@ def __init__(self, nick, user, host):
         self.channels = {}
         """The channels the user is in.
 
-        This maps channel name ``Identifier``\\s to ``Channel`` objects."""
+        This maps channel name :class:`Identifier <sopel.tools.Identifier>`\\s
```

Comment:
To achieve the same thing, you can add a `~ ` in front of the class path:
```suggestion
        This maps channel name :class:`~sopel.tools.Identifier`\\s
```

See the [sphinx documentation on cross-referencing syntax](https://www.sphinx-doc.org/en/2.0/usage/restructuredtext/roles.html#cross-referencing-syntax).

Reply:
Eh, this is what I get for inferring what `~` does instead of reading the docs. I thought it was a sort of "implied parent" like `~` in path names on \*nix, and referred to the current module. :rofl:

Reply:
Yeah, you have the `.` for something similar:

```
:class:`.ClassInTheCurrentModule`
```